Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2625,27 +2690,39 @@ def splunk_submit_event_hec(
     source_type: str | None,
     source: str | None,
     time_: str | None,
-    request_channel: str | None
+    request_channel: str | None,
+    batch_event_data: str | None,
+    entry_id: int | None,
+    service
 ):
     if hec_token is None:
         raise Exception('The HEC Token was not provided')
 
-    parsed_fields = None
-    if fields:
-        try:
-            parsed_fields = json.loads(fields)
-        except Exception:
-            parsed_fields = {'fields': fields}
+    if batch_event_data:
+        events = batch_event_data
 
-    args = assign_params(
-        event=event,
-        host=host,
-        fields=parsed_fields,
-        index=index,
-        sourcetype=source_type,
-        source=source,
-        time=time_
-    )
+    elif entry_id:
+        demisto.debug(f'Splunk {entry_id=}')
```

Comment:
```suggestion
        demisto.debug(f'Splunk - loading events data from file with {entry_id=}')
```
We should consider changing the `Splunk` to a Global param we can look for after that in the logs.